mh/pat_item.py

```diff
@@ -585,6 +585,51 @@ class UserSearchInfo(PatData):
         (0x10, "info_mine_0x10"),
     )
 
+class LayerPath(object):
+    FMT_WRITE = struct.Struct(">IIHHH")
```

Comment:
`STRUCT_FMT` (or `STRUCT`) would be a more fitting name since it's a Struct object that can be used to pack/unpack data and isn't limited to "write" operations.